test/integration/dashboard_test.rb

```diff
@@ -1,7 +1,7 @@
 require 'test_helper'
 
 class DashboardTest < ActionDispatch::IntegrationTest
-  def setup
+  before(:all) do
```

Comment:
Doesn't the DB reset between each test?  I'm not sure I understand when an :all would ever be useful.